src/utils/fs.ts

```diff
@@ -278,8 +278,18 @@ export function removeDependency (options: DefinitionOptions) {
 
   // Remove the dependency from typings.
   function remove (path: string, file: string, dtsFile: string) {
-    return promiseFinally(rimraf(path), () => {
-      return transformDtsFile(dtsFile, typings => typings.filter(x => x !== file))
+    const libName: string = path.split('/').slice(-1)[0]
+    const dtsType: string = dtsFile.split('/').slice(-1)[0]
+
+    isFile(file).then((flag) => {
+      if (!flag) {
+        return console.warn(`[${dtsType}] not installed in ${libName}`)
```

Comment:
We shouldn't use `console` in the programmatic API because we don't know where it'll be used. There's an emitter concept in `core` that might help with this PR.